src/components/Footer/Index.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect, useState } from 'react';
+import styles from './Index.module.scss';
+
+function Footer() {
+  const [date, setDate] = useState('');
+
+  const timerID = setInterval(() => setDate(new Date()), 1000);
```

Comment:
I'd say here on each render new timer gets created, but you never clear it anywhere. Memory leak ?